src/paho/mqtt/client.py

```diff
@@ -2595,58 +2873,60 @@ def _check_keepalive(self):
                 self._sock_close()
 
                 if self._state == mqtt_cs_disconnecting:
-                    rc = MQTT_ERR_SUCCESS
+                    rc = MQTTErrorCode.MQTT_ERR_SUCCESS
                 else:
-                    rc = MQTT_ERR_KEEPALIVE
+                    rc = MQTTErrorCode.MQTT_ERR_KEEPALIVE
 
                 self._do_on_disconnect(rc)
 
-    def _mid_generate(self):
+    def _mid_generate(self) -> int:
         with self._mid_generate_mutex:
             self._last_mid += 1
             if self._last_mid == 65536:
                 self._last_mid = 1
             return self._last_mid
 
     @staticmethod
-    def _topic_wildcard_len_check(topic):
+    def _topic_wildcard_len_check(topic: bytes) -> MQTTErrorCode:
```

Comment:
It would probably be more pythonic to just raise from here instead of return an error code (that's only ever used internally, one hopes) – or at least return a `bool`.